graphistry/collapse.py

```diff
@@ -0,0 +1,532 @@
+import pandas as pd
+import numpy as np
+
+COLLAPSE_NODE = "collapse_node"
+COLLAPSE_SRC = "collapse_src"
+COLLAPSE_DST = "collapse_dst"
+WRAP = "~"
+DEFAULT_VAL = "None"
+VERBOSE = True
+
+
+def unpack(g):
+    """
+        Helper method that unpacks graphistry instance
+    ex:
+        ndf, edf, src, dst, node = unpack(g)
+
+    -----------------------------------------------------------------------------------------
+
+    :param g: graphistry instance
+    :returns node DataFrame, edge DataFrame, source column, destination column, node column
+    """
+    ndf = g._nodes
+    edf = g._edges
+    src = g._source
+    dst = g._destination
+    node = g._node
+    return ndf, edf, src, dst, node
+
+
+def get_children(g, node_id, hops=1):
```

Comment:
unncessary as just calling hop?

Reply:
Yes and then I have to set it each time with a dataframe? Less writing?

Reply:
hmm maybe we can make `hop` signature more convenient then:

ex:  this could be replaced with an even short `g.hop([node_id], hops)`, where we teach `hop()` that if instead of nodes=pd.DataFrame, it gets `List[str]`, it auto-coerces into a nodes df for them

Reply:
so `hop(self, nodes=Union[ pd.DataFrame, List[str] ], ...`